### PR TITLE
fbp-to-dot: Abort if sol_init fails

### PR DESCRIPTION
**Pull request by anselmolsm**
Signed-off-by: Anselmo L. S. Melo <anselmo.melo@intel.com>

### CODE REVIEW

**Comment by ricardotk** [comment]
+1

File: src/bin/sol-fbp-to-dot/main.c [hunk 1]
```diff
@@ -350,5 +352,6 @@ main(int argc, char *argv[])
     free(fbpfile);
     free(dotfile);
     sol_shutdown();
+end:
     return 0;
```
Comment (anselmolsm) [inline src/bin/sol-fbp-to-dot/main.c]: But, of course this return should be != 0 in case of errors....

Comment (ricardotk) [inline src/bin/sol-fbp-to-dot/main.c]: Yes, I agree... see how generator or runner handles it

**Comment by anselmolsm** [comment]
v2 submitted: #231

**Pull request closed without merging**